### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by bghgary**

### CODE REVIEW

**Review by SergioRZMasson — APPROVED** [review]

**Review by PolygonalSun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
 1. Single view only - multiple views are not yet supported (only a single view can be displayed).
 
-It is also worth noting that Babylon React Native relies heavily on newer React Native constructs including JSI to get the performance characteristics required for real time rendering. JSI allows for direct synchronous communication between native code and JavaScript code, but is incompatible with "remote debugging." If you need to debug your JavaScript code that uses Babylon React Native, you should enable Hermes and use "direct debugging" (e.g. chrome://inspect or edge://inspect). See the [React Native documentation](https://reactnative.dev/docs/hermes) for more info.
+### What is Supported from Babylon.js
+
+See [this](https://github.com/BabylonJS/BabylonNative#what-is-supported-from-babylonjs) from the Babylon Native repo.
+
+### Performance
+
+React Native applications targeting [JavaScriptCore on iOS](https://reactnative.dev/docs/javascript-environment#javascript-runtime) or [Hermes](https://reactnative.dev/docs/hermes) will not have JIT enabled. This may cause performance issues for JavaScript heavy scenarios.
```
Comment (ryantrem) [inline README.md:22]: Even on Android I don't think there is any way to enable JIT with any JS engine.

File: README.md [hunk 2]
```diff
@@ -104,7 +113,7 @@ export PATH=$PATH:$ANDROID_HOME/tools/bin
 export PATH=$PATH:$ANDROID_HOME/platform-tools
 ```
 
-**Troubleshootings:**
+**Troubleshooting:**
```
Comment (ryantrem) [inline README.md:116]: We should point to the Babylon Native troubleshooting as well, including the stuff about Android emulators (we may even have an issue tracking this).

**Review by ryantrem — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bghgary) [reply README.md:22]: I don't think so either, but that's not what the React Native docs say, so I'm just repeating it.

File: README.md (hunk 2 above)
Comment (bghgary) [reply README.md:116]: Oh yeah, I forgot to add this.

Comment (bghgary) [reply README.md]: I added two tips, one pointing to Android Emulator directly and another for general Babylon Native docs.

**Pull request merged**